controller/lib/stl/new.h

```diff
@@ -21,7 +21,7 @@ limitations under the License.
 // has an STL.
 //
 // TODO: Remove this file once we no longer support AVR_UNO.
-#ifdef ARDUINO_AVR_UNO
+#if defined(ARDUINO_AVR_UNO) or defined(BARE_STM32)
```

Comment:
Does STM32 toolchain also not provide a placement new operator?

Reply:
OK, I have to admit that I don't use the STL very often.  I added this because I was getting a compile error.
It looks like we just need to #include <new> in blower_fsm.cpp.
Does that seem correct?  Why did other platforms compile without that include?

Reply:
That should have been #include \<new\>

Reply:
I think the right fix is to make `new.h` do

```
#if !defined(ARDUINO_AVR_UNO)
#include <new>
#else
inline void *operator new(size_t, void *__ptr) { return __ptr; }
#endif
```